project/cmake/scripts/linux/Install.cmake

```diff
@@ -132,6 +132,10 @@ install(FILES ${CORE_SOURCE_DIR}/copying.txt
         DESTINATION ${datarootdir}/doc/${APP_NAME_LC}
         COMPONENT kodi)
 
+install(FILES ${CORE_SOURCE_DIR}/privacy-policy.txt
+        DESTINATION ${datarootdir}${APP_NAME_LC}
+        COMPONENT kodi)
+
```

Comment:
special://xbmc/ points to ${datarootdir}${APP_NAME_LC}
(i need a special path that's the same across platforms)